test/unit/lib/utils/queryMapper.js

```diff
@@ -57,6 +57,17 @@ describe('queryMapper', () => {
       expect(output).to.equal(undefined);
     });
 
+    it('should return undefined when empty location', () => {
+      // eslint-disable-next-line no-shadow
```

Comment:
Rather than disabling eslint it would be clearer if a differently named variable was used.